tests/test_config.py

```diff
@@ -0,0 +1,29 @@
+from src.config import ConfigContainer, KMeans
+from src.file_handlers.json_handler import JsonHandler
+from src.file_handlers.numpy_handler import NumpyHandler
+
+from .data import CONFIG_DATA
+
+
+class TestConfigContainer:
+    def test_config_init(self, config_container: ConfigContainer):
```

Comment:
did you run the tests? I think they are not working.... It is a good practice to have test in CI/CD pipeline